mojaloop-payment-manager/Chart.yaml

```diff
@@ -29,7 +29,7 @@ dependencies:
     repository: https://charts.bitnami.com/bitnami
     condition: redis.enabled
   - name: sdk-scheme-adapter
-    version: 1.8.0
+    version: 1.9.0-18400-24b49e8
```

Comment:
This feels a bit weird. Maybe wait for a normal version here before merging?

Reply:
Yeah, but the problem is mojaloop helm repo follows its own release cycle and takes time to make a standard release. As p4ml is dependent on sdk helm chart there, lets proceed with this devlopment release.